lib/models/Message.js

```diff
@@ -109,6 +109,45 @@ export default class Message extends Model {
   }
 
   /**
+   * Updates the isFlagged field of this {@link Message}.
+   *
+   * @param [isFlagged=false] {Boolean} The isFlagged field of the message.
+   *
+   * @return {Promise} A {@link Promise}, as per {@link Client.updateMessage}.
+   *
+   * @see Client#updateMessage
+   */
+  setIsFlagged(isFlagged) {
+    return this._jmap.updateMessage(this.id, { isFlagged: isFlagged || false });
```

Comment:
I don't think that we should make `isFlagged` optional.
It'll never be used in this way `message.isFlagged()`, right? so having `isFlagged` undefined there come from a bug IMO.

I prefer that my lib tells me that something goes wrong instead of doing non-smart decision itself. I'd use `Utils.assertRequiredParameterIsPresent(isFlagged, 'isFlagged');` there!